src/async_impl/client.rs

```diff
@@ -758,6 +763,7 @@ impl ClientBuilder {
     /// This requires the optional `native-tls` or `rustls-tls(-...)` feature to be
     /// enabled.
     #[cfg(any(feature = "native-tls", feature = "__rustls"))]
+    #[cfg_attr(docsrs, doc(cfg(any(feature = "default-tls", feature = "native-tls", feature = "rustls-tls"))))]
```

Comment:
hmm, i had in my mind that the features were defined the other way around.

Reply:
I can't recall why *this* one claims to need either `native-tls` or `rustls-tls` to be enabled, but I did expect that features that might exist only in `native-tls` wouldn't be enabled by `default-tls`, in case it's possible to change the default to rustls at some point...